lib/membrane/rtp/outbound_packet_tracker.ex

```diff
@@ -11,18 +11,23 @@ defmodule Membrane.RTP.OutboundPacketTracker do
   require Membrane.Logger
   alias Membrane.RTCP.FeedbackPacket.{FIR, PLI}
   alias Membrane.RTP.Session.SenderReport
-  alias Membrane.{Buffer, Payload, RTCPEvent, RTP, Time}
+  alias Membrane.{Buffer, Payload, RemoteStream, RTCPEvent, RTP, Time}
 
-  def_input_pad :input, caps: :any, demand_mode: :auto
+  def_input_pad :input, caps: RTP, demand_mode: :auto
 
-  def_output_pad :output, caps: :any, demand_mode: :auto
+  def_output_pad :output,
+    caps: {RemoteStream, type: :packetized, content_format: RTP},
+    demand_mode: :auto
```

Comment:
It doesn't seem really intuitive to have `Tracker` that changes caps, especially since it doesn't modify the packets.
I think keeping it as RTP would allow SRT(C)P encoders to use RTP caps as well.